### PR TITLE
ppanggolin: add aarch64 build

### PR DESCRIPTION
**Pull request by martin-g**
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

### General instructions

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

### Instructions for avoiding API, ABI, and CLI breakage issues
Conda is able to record and lock (a.k.a. pin) dependency versions used at build time of other recipes.
This way, one can avoid that expectations of a downstream recipe with regards to API, ABI, or CLI are violated by later changes in the recipe.
If not already present in the meta.yaml, make sure to specify `run_exports` (see [here](https://bioconda.github.io/contributor/linting.html#missing-run-exports) for the rationale and comprehensive explanation).
Add a `run_exports` section like this:

```yaml
build:
  run_exports:
    - ...

```

with `...` being one of:

| Case                             | run_exports statement                                               |
| -------------------------------- | ------------------------------------------------------------------- |
| semantic versioning              | `{{ pin_subpackage("myrecipe", max_pin="x") }}`     |
| semantic versioning (0.x.x)      | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}`   |
| known breakage in minor versions | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| known breakage in patch versions | `{{ pin_subpackage("myrecipe", max_pin="x.x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| calendar versioning              | `{{ pin_subpackage("myrecipe", max_pin=None) }}`    |

while replacing `"myrecipe"` with either `name` if a `name|lower` variable is defined in your recipe or with the lowercase name of the package in quotes.

### Bot commands for PR management

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

Note that the <code>@BiocondaBot please merge</code> command is now depreciated. Please just squash and merge instead.

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/ppanggolin/meta.yaml [hunk 1]
```diff
@@ -55,6 +54,9 @@ about:
   summary: "PPanGGOLiN: Depicting microbial species diversity via a Partitioned PanGenome Graph"
 
 extra:
+  additional-platforms:
+    - linux-aarch64
+    - osx-arm64
```
Comment (martin-g) [inline recipes/ppanggolin/meta.yaml:59]:
```suggestion
```

**Comment by martin-g** [comment]
@BiocondaBot please fetch artifacts

**Comment by martin-g** [comment]
```
mgrigorov in 🌐 euler-arm-22 in /tmp/ppaggolin via 🅒 test123 took 8s 
❯ tar xvf ppanggolin-2.1.2-py310h8c4ab44_1.tar.bz2                                                                                                                                                         (test123) 
info/run_exports.json
info/test/run_test.py
info/test/run_test.sh
info/recipe/build.sh
info/hash_input.json
info/has_prefix
info/index.json
info/recipe/meta.yaml.template
info/recipe/meta.yaml
info/recipe/conda_build_config.yaml
info/about.json
info/files
info/paths.json
info/git
lib/python3.10/site-packages/PPanGGOLiN-2.1.2.dist-info/INSTALLER
lib/python3.10/site-packages/PPanGGOLiN-2.1.2.dist-info/WHEEL
lib/python3.10/site-packages/PPanGGOLiN-2.1.2.dist-info/RECORD
lib/python3.10/site-packages/PPanGGOLiN-2.1.2.dist-info/METADATA
lib/python3.10/site-packages/PPanGGOLiN-2.1.2.dist-info/REQUESTED
lib/python3.10/site-packages/ppanggolin/nem/NEM/nem_stats.pyx
lib/python3.10/site-packages/ppanggolin/annotate/rRNA_DB/rRNA_bact.cm.i1p
lib/python3.10/site-packages/ppanggolin/annotate/rRNA_DB/rRNA_arch.cm.i1p
bin/ppanggolin
lib/python3.10/site-packages/PPanGGOLiN-2.1.2.dist-info/top_level.txt
lib/python3.10/site-packages/PPanGGOLiN-2.1.2.dist-info/entry_points.txt
lib/python3.10/site-packages/PPanGGOLiN-2.1.2.dist-info/LICENSE.txt
lib/python3.10/site-packages/ppanggolin/nem/NEM/nem_ver.h
lib/python3.10/site-packages/ppanggolin/nem/NEM/nem_nei.h
lib/python3.10/site-packages/ppanggolin/nem/NEM/nem_hlp.h
lib/python3.10/site-packages/ppanggolin/nem/NEM/nem_rnd.h
lib/python3.10/site-packages/ppanggolin/nem/NEM/nem_exe.h
lib/python3.10/site-packages/ppanggolin/nem/NEM/genmemo.h
lib/python3.10/site-packages/ppanggolin/nem/NEM/nem_mod.h
lib/python3.10/site-packages/ppanggolin/nem/NEM/nem_alg.h
lib/python3.10/site-packages/ppanggolin/nem/NEM/lib_io.h
lib/python3.10/site-packages/ppanggolin/nem/NEM/nem_typ.h
lib/python3.10/site-packages/ppanggolin/annotate/rRNA_DB/rRNA_bact.cm.i1m
lib/python3.10/site-packages/ppanggolin/annotate/rRNA_DB/rRNA_arch.cm.i1m
lib/python3.10/site-packages/ppanggolin/meta/__init__.py
lib/python3.10/site-packages/ppanggolin/info/__init__.py
lib/python3.10/site-packages/ppanggolin/utility/__init__.py
lib/python3.10/site-packages/ppanggolin/mod/__init__.py
lib/python3.10/site-packages/ppanggolin/metrics/__init__.py
lib/python3.10/site-packages/ppanggolin/cluster/__init__.py
lib/python3.10/site-packages/ppanggolin/figures/__init__.py
lib/python3.10/site-packages/ppanggolin/annotate/__init__.py
lib/python3.10/site-packages/ppanggolin/projection/__init__.py
lib/python3.10/site-packages/ppanggolin/graph/__init__.py
lib/python3.10/site-packages/ppanggolin/align/__init__.py
lib/python3.10/site-packages/ppanggolin/context/__init__.py
lib/python3.10/site-packages/ppanggolin/RGP/__init__.py
lib/python3.10/site-packages/ppanggolin/workflow/__init__.py
lib/python3.10/site-packages/ppanggolin/formats/__init__.py
lib/python3.10/site-packages/ppanggolin/workflow/panRGP.py
lib/python3.10/site-packages/ppanggolin/workflow/panModule.py
lib/python3.10/site-packages/ppanggolin/workflow/workflow.py
lib/python3.10/site-packages/ppanggolin/__init__.py
lib/python3.10/site-packages/ppanggolin/figures/ucurve.py
lib/python3.10/site-packages/ppanggolin/edge.py
lib/python3.10/site-packages/ppanggolin/metrics/fluidity.py
lib/python3.10/site-packages/ppanggolin/info/info.py
lib/python3.10/site-packages/ppanggolin/figures/drawing.py
lib/python3.10/site-packages/ppanggolin/metrics/metrics.py
lib/python3.10/site-packages/ppanggolin/graph/makeGraph.py
lib/python3.10/site-packages/ppanggolin/formats/writeFlatMetadata.py
lib/python3.10/site-packages/ppanggolin/mod/module.py
lib/python3.10/site-packages/ppanggolin/utility/utils.py
lib/python3.10/site-packages/ppanggolin/main.py
lib/python3.10/site-packages/ppanggolin/metadata.py
lib/python3.10/site-packages/ppanggolin/meta/meta.py
lib/python3.10/site-packages/ppanggolin/RGP/spot.py
lib/python3.10/site-packages/ppanggolin/formats/write_proksee.py
lib/python3.10/site-packages/ppanggolin/RGP/genomicIsland.py
lib/python3.10/site-packages/ppanggolin/annotate/synta.py
lib/python3.10/site-packages/ppanggolin/formats/writeMetadata.py
lib/python3.10/site-packages/ppanggolin/geneFamily.py
lib/python3.10/site-packages/ppanggolin/figures/tile_plot.py
lib/python3.10/site-packages/ppanggolin/formats/writeMSA.py
lib/python3.10/site-packages/ppanggolin/formats/writeAnnotations.py
lib/python3.10/site-packages/ppanggolin/workflow/all.py
lib/python3.10/site-packages/ppanggolin/RGP/rgp_cluster.py
lib/python3.10/site-packages/ppanggolin/nem/rarefaction.py
lib/python3.10/site-packages/ppanggolin/figures/draw_spot.py
lib/python3.10/site-packages/ppanggolin/formats/writeFlatGenomes.py
lib/python3.10/site-packages/ppanggolin/formats/writeSequences.py
lib/python3.10/site-packages/ppanggolin/cluster/cluster.py
lib/python3.10/site-packages/ppanggolin/formats/writeBinaries.py
lib/python3.10/site-packages/ppanggolin/context/searchGeneContext.py
lib/python3.10/site-packages/ppanggolin/align/alignOnPang.py
lib/python3.10/site-packages/ppanggolin/pangenome.py
lib/python3.10/site-packages/ppanggolin/nem/partition.py
lib/python3.10/site-packages/ppanggolin/genome.py
lib/python3.10/site-packages/ppanggolin/region.py
lib/python3.10/site-packages/ppanggolin/formats/readBinaries.py
lib/python3.10/site-packages/ppanggolin/utils.py
lib/python3.10/site-packages/ppanggolin/formats/writeFlatPangenome.py
lib/python3.10/site-packages/ppanggolin/annotate/annotate.py
lib/python3.10/site-packages/ppanggolin/projection/projection.py
lib/python3.10/site-packages/ppanggolin/nem/__init__.py
lib/python3.10/site-packages/ppanggolin/genetic_codes.py
lib/python3.10/site-packages/ppanggolin/nem/NEM/nem_ver.c
lib/python3.10/site-packages/ppanggolin/nem/NEM/genmemo.c
lib/python3.10/site-packages/ppanggolin/nem/NEM/nem_rnd.c
lib/python3.10/site-packages/ppanggolin/nem/NEM/nem_nei.c
lib/python3.10/site-packages/ppanggolin/nem/NEM/lib_io.c
lib/python3.10/site-packages/ppanggolin/nem/NEM/nem_hlp.c
lib/python3.10/site-packages/ppanggolin/nem/NEM/nem_exe.c
lib/python3.10/site-packages/ppanggolin/nem/NEM/nem_mod.c
lib/python3.10/site-packages/ppanggolin/nem/NEM/nem_alg.c
lib/python3.10/site-packages/nem_stats.cpython-310-aarch64-linux-gnu.so
lib/python3.10/site-packages/PPanGGOLiN-2.1.2.dist-info/direct_url.json
lib/python3.10/site-packages/ppanggolin/annotate/rRNA_DB/RF00001.cm
lib/python3.10/site-packages/ppanggolin/annotate/rRNA_DB/RF01959.cm
lib/python3.10/site-packages/ppanggolin/annotate/rRNA_DB/RF00177.cm
lib/python3.10/site-packages/ppanggolin/annotate/rRNA_DB/RF02541.cm
lib/python3.10/site-packages/ppanggolin/annotate/rRNA_DB/RF02540.cm
lib/python3.10/site-packages/ppanggolin/annotate/rRNA_DB/rRNA_bact.cm
lib/python3.10/site-packages/ppanggolin/annotate/rRNA_DB/rRNA_arch.cm
lib/python3.10/site-packages/ppanggolin/nem/__pycache__/__init__.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/meta/__pycache__/__init__.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/info/__pycache__/__init__.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/mod/__pycache__/__init__.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/utility/__pycache__/__init__.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/metrics/__pycache__/__init__.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/figures/__pycache__/__init__.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/graph/__pycache__/__init__.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/cluster/__pycache__/__init__.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/align/__pycache__/__init__.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/annotate/__pycache__/__init__.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/projection/__pycache__/__init__.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/context/__pycache__/__init__.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/RGP/__pycache__/__init__.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/workflow/__pycache__/__init__.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/formats/__pycache__/__init__.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/workflow/__pycache__/panRGP.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/workflow/__pycache__/panModule.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/workflow/__pycache__/workflow.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/__pycache__/__init__.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/figures/__pycache__/ucurve.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/metrics/__pycache__/fluidity.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/figures/__pycache__/drawing.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/info/__pycache__/info.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/__pycache__/edge.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/metrics/__pycache__/metrics.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/graph/__pycache__/makeGraph.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/formats/__pycache__/writeFlatMetadata.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/__pycache__/main.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/utility/__pycache__/utils.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/mod/__pycache__/module.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/meta/__pycache__/meta.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/formats/__pycache__/write_proksee.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/formats/__pycache__/writeMetadata.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/__pycache__/metadata.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/RGP/__pycache__/spot.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/RGP/__pycache__/genomicIsland.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/workflow/__pycache__/all.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/annotate/__pycache__/synta.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/figures/__pycache__/tile_plot.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/formats/__pycache__/writeMSA.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/formats/__pycache__/writeAnnotations.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/__pycache__/geneFamily.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/nem/__pycache__/rarefaction.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/formats/__pycache__/writeBinaries.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/figures/__pycache__/draw_spot.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/formats/__pycache__/writeFlatGenomes.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/cluster/__pycache__/cluster.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/align/__pycache__/alignOnPang.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/context/__pycache__/searchGeneContext.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/nem/__pycache__/partition.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/formats/__pycache__/writeSequences.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/RGP/__pycache__/rgp_cluster.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/__pycache__/pangenome.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/formats/__pycache__/readBinaries.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/__pycache__/genome.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/__pycache__/region.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/__pycache__/utils.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/annotate/__pycache__/annotate.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/formats/__pycache__/writeFlatPangenome.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/projection/__pycache__/projection.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/__pycache__/genetic_codes.cpython-310.pyc
lib/python3.10/site-packages/ppanggolin/annotate/rRNA_DB/rRNA_bact.cm.i1f
lib/python3.10/site-packages/ppanggolin/annotate/rRNA_DB/rRNA_arch.cm.i1f
lib/python3.10/site-packages/ppanggolin/annotate/rRNA_DB/rRNA_arch.cm.i1i
lib/python3.10/site-packages/ppanggolin/annotate/rRNA_DB/rRNA_bact.cm.i1i

mgrigorov in 🌐 euler-arm-22 in /tmp/ppaggolin via 🅒 test123 
❯ file bin/ppanggolin                                                                                                                                                                                      (test123) 
bin/ppanggolin: POSIX shell script, ASCII text executable

mgrigorov in 🌐 euler-arm-22 in /tmp/ppaggolin via 🅒 test123 
❯ find ./ -iname "*.so"                                                                                                                                                                                    (test123) 
./lib/python3.10/site-packages/nem_stats.cpython-310-aarch64-linux-gnu.so

mgrigorov in 🌐 euler-arm-22 in /tmp/ppaggolin via 🅒 test123 
❯ file ./lib/python3.10/site-packages/nem_stats.cpython-310-aarch64-linux-gnu.so                                                                                                                           (test123) 
./lib/python3.10/site-packages/nem_stats.cpython-310-aarch64-linux-gnu.so: ELF 64-bit LSB shared object, ARM aarch64, version 1 (SYSV), dynamically linked, not stripped
```
LGTM!

**Comment by martin-g** [comment]
@BiocondaBot please add label

**Review by mencian — APPROVED** [review]

**Pull request merged**